src/main/resources/org/opentripplanner/apis/transmodel/schema.graphql

```diff
@@ -2092,6 +2097,58 @@ input PenaltyForStreetMode {
   timePenalty: DoubleFunction!
 }
 
+"""
+One of the listed stop locations must be visited on-board a transit vehicle or the journey must
+alight or board at the location.
```

Comment:
Was it possible with pass through points that you walk from the origin to a passthrough stop and board from there (not just from walking from a transfer stop)?

Reply:
Yes, https://github.com/opentripplanner/OpenTripPlanner/blob/fb744f00f55c510bca5719ccc320584c4b4c6bd3/src/test/java/org/opentripplanner/raptor/moduletests/J01_PassThroughTest.java#L120

Reply:
Yes, it is only the coordinates witch has this limitation.